starting_from_scratch_lab/lab_40/tar2.js

```diff
@@ -0,0 +1,5 @@
+const selectElement = document.querySelector(".source");
+selectElement.addEventListener("change", (event) => {
+  var inputs = document.querySelectorAll("input");
```

Comment:
Since the input doesn't change, it is better to define them outside of the function (and change `var` to `const`) to prevent them from being recreated again and again. It has little importance at this level of complexity, but might cause issues in bigger projects.